wiki/People/Nomination_Assessment_Team/en.md

```diff
@@ -96,64 +126,76 @@ The [Nomination Assessment Team group page](https://osu.ppy.sh/groups/7) lists a
+| ::{ flag=BR }:: [Ideal](https://osu.ppy.sh/users/3869519) | Portuguese |
+
+#### osu!catch
+
+| Name | Additional languages |
+| :-- | :-- |
+| ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565) | Spanish, German |
+| ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776) | Dutch |
+| ::{ flag=US }:: [Secre](https://osu.ppy.sh/users/2306637) |  |
+
+#### osu!mania
+
+| Name | Additional languages |
+| :-- | :-- |
+| ::{ flag=VN }:: [Akasha-](https://osu.ppy.sh/users/2596306) | Vietnamese |
+| ::{ flag=CN }:: [gzdongsheng](https://osu.ppy.sh/users/8660315) | Chinese |
+| ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) | Indonesian |
+| ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) | Spanish, Catalan |
+| ::{ flag=DK }:: [Tailsdk](https://osu.ppy.sh/users/6751666) | Danish |
 
-In addition, the [Nomination Assessment Team](https://osu.ppy.sh/users/6616586) bot acts as a proxy for the NAT to collectively deliver messages under one user account without directly attributing it to anyone. It may be used for evaluation responses, mock evaluations, mediations, reports, and content reviews, along any other miscellaneous purposes.
+### Structural
+
+The following NAT members are exclusively responsible for structural tasks, and do not associate with any specific game mode, despite being allowed to nominate beatmap from their previous mode.
 
-## Notes
+- ::{ flag=GB }:: [-Mo-](https://osu.ppy.sh/users/2202163): Handling internal discussions, affairs, and documentation
+- ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817): [Aiess](https://github.com/Naxesss/Aiess) and [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier) maintenance
+- ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418): [NAT/BN website](https://bn.mappersguild.com/home) development
+- ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099): NAT leadership
 
-[^task-fuju]: Mainly handles user reports and managing SEV ratings for the osu! game mode
-[^task-mo]: Mainly handles internal discussions, affairs, and documentation
-[^task-naxess]: Mainly maintains [Aiess](https://github.com/Naxesss/Aiess) and [Mapset Verifier](https://github.com/Naxesss/MapsetVerifier)
-[^task-pishifat]: Mainly maintains [the NAT/BN website](https://bn.mappersguild.com/home)
-[^task-star]: Mainly maintains the ranking criteria test questions
+The following NAT members are primarily evaluators, but also contribute to structural tasks:
+
+- ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507): BN Mentorship organisation
+- ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882): Handling user reports and osu! SEV ratings
+- ::{ flag=CN }:: [Garden](https://osu.ppy.sh/users/2849992): Handling content reports
+- ::{ flag=NL }:: [Greaper](https://osu.ppy.sh/users/2369776): [Nomination Planner](https://bnplanner.greaper.net/beatmaps) and [osu!catch MV plugin](https://github.com/Darius-Wattimena/MapsetChecksCatch) development
```

Comment:
are these supposed to stay as plugins? i'd rather just put "Mapset Verifier development" because they sound like temporary tasks

Reply:
they've always been external plugins and i doubt they're getting added anytime soon

i'll make sure to update this once (if) they actually become a part of the core MV installation